README.md

```diff
@@ -57,11 +57,15 @@ from django.conf import settings
 from apscheduler.schedulers.blocking import BlockingScheduler
 from apscheduler.triggers.cron import CronTrigger
 from django.core.management.base import BaseCommand
-from django_apscheduler.jobstores import DjangoJobStore
+from django_apscheduler.jobstores import DjangoJobStore, register_events
 from django_apscheduler.models import DjangoJobExecution
 
 
 logger = logging.getLogger(__name__)
+console = logging.StreamHandler()
+console.setLevel(logging.INFO)
+logger.addHandler(console)
+logger.setLevel(logging.INFO)
```

Comment:
I think users should be setting this up themselves in their django settings file. They can elect which handlers they want and which levels they want them logging at.

Reply:
These settings will only effect the `runapscheduler.py` file and if you only get a logger, `logger.info` might not work work as what you want.

Reply:
`runapscheduler.py` is a Django management command, which I ***think*** loads `DJANGO_SETTINGS_MODULE` by default. If Django logging settings have been configured correctly then it shouldn't be necessary to re-define a logger here?